packages/flutter_tools/test/general.shard/test/web_test_compiler_test.dart

```diff
@@ -0,0 +1,94 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file/memory.dart';
+import 'package:flutter_tools/src/artifacts.dart';
+import 'package:flutter_tools/src/base/config.dart';
+import 'package:flutter_tools/src/base/logger.dart';
+import 'package:flutter_tools/src/base/platform.dart';
+import 'package:flutter_tools/src/build_info.dart';
+import 'package:flutter_tools/src/test/web_test_compiler.dart';
+import 'package:test/expect.dart';
+
+import '../../src/context.dart';
+
+void main() {
+  testUsingContext('web test compiler issues valid compile command', () async {
```

Comment:
Consider leaving the link to the issue so a future reader can understand what needs to be valid about the compile command.